contents/handbook/making-users-happy.md

```diff
@@ -0,0 +1,27 @@
+---
+title: How we make users happy
+sidebar: Handbook
+showTitle: true
+---
+
+## How we build a product
+
+First, someone internally will suggest an idea. Usually this will come from James and Tim, but it has just as frequently come from anyone else on the team.
```

Comment:
```suggestion
First, someone internally will suggest an idea. Sometimes this will come from James and Tim, but it has just as frequently come from anyone else on the team.
```